_api-reference/cat/cat-allocation.md

```diff
@@ -51,7 +51,7 @@ In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-refe
 Parameter | Type | Description
 :--- | :--- | :---
 bytes | Byte size | Specify the units for byte size. For example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
-local | Boolean | Whether to return information from the local node only instead of from the cluster_manager node. Default is false.
+local | Boolean | Whether to return information from the local node only instead of from the cluster_manager node. Default is `false`.
```

Comment:
```suggestion
local | Boolean | Whether to return information from the local node only instead of from the cluster manager node. Default is `false`.
```